lecture_4/03_bet_acceptance_api/cmd/bootstrap/rabbitmq.go

```diff
@@ -0,0 +1,18 @@
+package bootstrap
+
+import "github.com/streadway/amqp"
+
+// RabbitMq bootstraps the rabbit mq connection.
+func RabbitMq() *amqp.Channel {
+	conn, err := amqp.Dial("amqp://guest:guest@localhost:5672/")
```

Comment:
Message queue connection string could also be part of the configuration instead of being hardcoded.